### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by StephenRiceMSFT**
Adding stub for new parameter in Set-SPOTenant.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1679,6 +1680,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -BlockUserInfoVisibility
+> [!NOTE]
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1684]: Remove ![Note] as it cannot be parsed in PowerShell.

**Comment by get-itips** [comment]
agree when Trevor's suggestion is applied

**Comment by StephenRiceMSFT** [comment]
@tseward agree on this change. Little unfamiliar with GitHub so unsure how to submit the change.

**Comment by tseward** [comment]
@StephenRiceMSFT make another edit to your commit :)

**Comment by StephenRiceMSFT** [comment]
@tseward any other changes need to be made before we can publish this?

**Comment by tseward** [comment]
@StephenRiceMSFT you're good to go to merge! :-)

**Comment by StephenRiceMSFT** [comment]
@tseward I may be blind, not seeing the merge button :) Do I just need to close the pull request?

**Comment by tseward** [comment]
@StephenRiceMSFT OK I see the same thing and this should have reached production by now. I even see that @Kateyanne had committed with your changes.

All I can suggest is to create a new PR with the detailed information about the parameter. Sorry I can't be of more help :(

**Comment by StephenRiceMSFT** [comment]
Thanks @tseward! I must admit to being even more confused now... I just went to edit the file & the GitHub version actually has my text change from this commit. The parameters also show up in a different order than in the final page as well. I'm honestly at a loss as to how to proceed here.

**Comment by tseward** [comment]
@StephenRiceMSFT OK, so if you don't have any other PRs waiting to be merged, you can delete your local copy of the repo (https://github.com/StephenRiceMSFT/office-docs-powershell) and then make a new PR.

**Pull request merged**